test/common/testlib.py

```diff
@@ -1241,6 +1271,18 @@ def wrapper(*args):
     return wrapper
 
 
+def timeout(seconds):
+    """Change default test timeout of 300s, for long running tests
```

Comment:
Here you mention 300s default, but in `armTestTimeout` you talk about 10 minutes and also have default 600s.
I would prefer 5 minutes though, 10 is too much. Are there many tests that are close to 10 minutes?

Reply:
We have relatively few which take between 150 and 270s, i. e. dangerously close to 5 mins, especially if they run on slower infra. I wouldn't want to endanger tests with a too rigid timeout. The main point of this is to avoid tests hanging around for 2 hours, so that we have to kill them manually to see what's wrong, and to avoid long waiting times to find out about them in the first place. I think 10 mins is an acceptable delay.

Indeed I started with 5mins when I wrote that, and updated it to 10 yesterday when analyzing some logs. I just forgot to update this comment.